cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -215,6 +232,33 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
         return points;
     }
 
+    private runImageModifier():void {
+        if (this.activeImageModifier) {
+            const {
+                data, states, curZOrder, canvasInstance, frame,
+            } = this.props;
+            const canvas: HTMLCanvasElement | undefined = window.document.getElementById('cvat_canvas_background') as
+                | HTMLCanvasElement
+                | undefined;
+            if (!canvas) {
+                throw new Error('Element #cvat_canvas_background was not found');
+            }
+            const { width, height } = canvas;
+            const context = canvas.getContext('2d');
+            if (!context) {
+                throw new Error('Canvas context is empty');
+            }
+            const imageData = context.getImageData(0, 0, width, height);
+            this.activeImageModifier.equalize(imageData, frame).then((newBitmap:ImageBitmap|undefined) => {
+                if (newBitmap) {
+                    // eslint-disable-next-line no-underscore-dangle
+                    data._data.imageData = newBitmap;
+                    canvasInstance.setup(data, states, curZOrder, true);
+                }
+            });
+        }
+    }
```

Comment:
Generally, several modifiers are supposed. In this method, I would suggest receiving an image from the canvas, running all modifiers one by another for the image in a chain, and setting the image back to the canvas

Current implementation is very similar, but it works with only one modifier

Reply:
Done